build/teamcity/cockroach/ci/tests/unit_tests_impl.sh

```diff
@@ -2,6 +2,10 @@
 
 set -xeuo pipefail
 
+dir="$(dirname $(dirname $(dirname $(dirname $(dirname "${0}")))))"
+
+source "$dir/teamcity-support.sh"  # For $root (defines 'tc_release_branch')
```

Comment:
Nit: `# for tc_release_branch` makes more sense to me as a comment here, since we're not actually using `$root`.

Reply:
Copy/pasta, fixed.